include/grpcpp/impl/codegen/completion_queue.h

```diff
@@ -365,9 +365,7 @@ class ServerCompletionQueue : public CompletionQueue {
  public:
   bool IsFrequentlyPolled() { return polling_type_ != GRPC_CQ_NON_LISTENING; }
 
- private:
-  grpc_cq_polling_type polling_type_;
-  friend class ServerBuilder;
+ protected:
```

Comment:
Ok, we have a core leakage again. Sorry for not catching this earlier. By promoting the `ServerCompletionQueue` constructor to `protected`, it has become API. However, this API has an argument that is a core enum. Can you look at your callsites and perhaps just elevate a constructor for a specific case to protected (since I assume that you have a subset of the possible polling type choices) and make that a bool argument or something that then delegates something else to the pre-existing private constructor?

Reply:
done